pytealutils/storage/blob/blob.py

```diff
@@ -0,0 +1,211 @@
+from typing import Tuple
```

Comment:
Is this extra `blob` directory needed?
Seems like you could just have the `blob.py` right under `storage/`

Reply:
Definitely could, would that be more appropriate?

Reply:
Since `blob.py` is self contained, it's simpler to move it up. If in the future it gets more complicated and has a bunch of helpers, you can move it into its own directory